File: Frontend/src/app/models/GetTasksDto.ts
export class GetTasksDto{
    
    ticketID:string;
    description:string;
    status:boolean;
    date:Date;

    constructor(TickedID:string, Description:string, Status:boolean, dateString: string)
    {
        this.ticketID = TickedID;
        this.description = Description;
        this.status = Status;
        this.date = new Date(dateString);
    }

}

File: Backend/Coding-Game-Backend/Models/DTOs/getTasksDto.cs
using System.ComponentModel.DataAnnotations;

namespace Coding_Game_Backend.Models.DTOs
{
    public class getTasksDto
    {
        public int TicketID {  get; set; }
        public string Description { get; set; } = string.Empty;
        public bool Status { get; set; }
        public DateTime Date { get; set; }
    }
}
